### DISCUSSION TITLE
I changed from version 2.4.1 to 3.0.0 or higher and I have not been able to run the tests again

### DISCUSSION THREAD

**Discussion by whitebeareth** [Synthetixio/synpress#591] [Q&A] [answered]
Hey Guys! 
I had the project running correctly in version 2.4.1, I updated it to the latest version or to any version 3.0.0 onwards to be able to use the scroll, and the small down arrow for the signatures, among others, but I can't make it run again.

I have environment variables in the system and also in a .env file. I tried different ways and commands, but it always ends up in the Synpress.JS file showing a different error: 
- 'Skipping check for SECRET_WORDS and PRIVATE_KEY as SKIP_METAMASK_INSTALL or SKIP_METAMASK_SETUP is set'.

-Please provide SECRET_WORDS or PRIVATE_KEY environment variable'.

Or other similar.

Can anyone help me with what command format to send or how to make it take the installation, setup, secret phrase, etc., in this new version?

**Comment by drptbl** [comment]
Hey,
Is your code open source?

How can I reproduce your issue?

Thanks,
Jakub.

**Reply by whitebeareth** [reply]
Hey drptbl!

First of all it is important for you to know that I am using WINDOWS.

I can't share mine, but I was able to replicate the error from one that is already published, which is this one:
https://github.com/iQaiserAbbas/dafi-automation-synpress 

Once downloaded:
1-I follow the steps in the README of that file.
2- install yarn
3-Then update to version 3.0.0 or later.
4-Add the file synpress.js to the root and in the scripts I change the path from synpres.json to synpress.js.
5-  Execute npm run test / yarn run test

This one brings me for example one of so many errors:
![image](https://user-images.githubusercontent.com/113952458/205653884-50c6bcbd-5e59-4c8f-9c6b-41704a115254.png)

**Comment by whitebeareth** [comment]
Hey @drptbl !
Some progress in solving the issue.
I use Windows so it did not accept the variables in the format that is in the readme, so I installed cross-env and now it does not give me any errors that before it gave me... but I'm stuck with this error:

![image](https://user-images.githubusercontent.com/113952458/210472808-a21e5103-60b0-4c90-b219-3cc9561bc79d.png)


I don't understand why, since I pass the following command:

npx cross-env SECRET_WORDS='xxxx, xxxx, xxxx, xxxx, xxxx, xxxx, xxxx, xxxx, xxxx, xxxx, xxxx, xxxx' synpress run --config baseUrl=https://staking.synthetix.io/ --browser chrome

**Comment by whitebeareth** [comment, marked as answer]
Great news @drptbl  ! I was able to run Synpress version 3.0.5 + MM 10.21.0 on Windows.
For those who have the same problem I recommend you to use this repo by @Belf33 :
https://github.com/Belf33/Cynpress_Metamask_Mvp 

Follow the instructions there, upgrade synpress version and change the .env.example file to the network and data you want to use🦾
